sourcegraph/cody-vs
# Autocomplete

- implemented the agent protocols for autocompletions
- inline text changes that can be accepted with the `Tab` keyboard press (`lalt+,` for rotating suggestions if there is more than one)
- explicitly trigger completion suggestions (`lalt+.`)
- user option to enable/disable Autocompletions
- telemetry to compute CAR
- tests

## Test plan
- Create a new solution
- Start adding code
- Check if 'grey code' with suggestions appears
- Accept the suggestions by pressing `Tab`
- Reject the suggestion by pressing `Esc`
